sections/semantics-forms.include

```diff
@@ -1209,7 +1209,7 @@
   <pre highlight="html">
 &lt;form&gt;
   &lt;table&gt;
-    &lt;caption&gt;Example &lt;<code>label</code>:gt;'s <code>for</code> attribute with &lt;<code>label</code>&gt;&lt;/caption&gt;
+    &lt;caption&gt;Example &lt;label:gt;'s for attribute with &lt;label&gt;&lt;/caption&gt;
```

Comment:
It would be nice to have a colon after example, and "with label" seems totally redundant.

(Yes, this is nitpicking editorial comment…)